plugin-flex-ts-template-v2/src/feature-library/chat-to-video-escalation/custom-components/SwitchToVideo/SwitchToVideo.tsx

```diff
@@ -77,7 +77,7 @@ const SwitchToVideo: React.FunctionComponent<SwitchToVideoProps> = ({ task, conv
         }
 
         Actions.invokeAction('SendMessage', {
-          body: templates[StringTemplates.InviteMessage]({ videoLink: response.full_url }),
+          body: `${templates[StringTemplates.InviteMessage]()}: ${response.full_url}`,
```

Comment:
I think the `:` should be part of the translate-able string because some languages format that intent differently.